luigi_pipeline/seqr_loading.py

```diff
@@ -118,7 +118,7 @@ def read_input_write_mt(self):
         kwargs = self.get_schema_class_kwargs()
         mt = self.SCHEMA_CLASS(mt, **kwargs).annotate_all(overwrite=True).select_annotated_mt()
 
-        mt = mt.annotate_globals(sourceFilePath=','.join(self.source_paths),
+        mt = mt.select_globals(sourceFilePath=','.join(self.source_paths),
```

Comment:
why is this necessary? The point is to share code where possible, NOT to change the current SNP pipeline behavior. Unless there is an incredibly compelling reason to make this change, the existing pipeline needs to stay as is. If there is a very compelling reason that the mito pipeline needs this to be a different function, then you need to make a helper function and override it for mito. But I really do think we want them to share code as much as possible, so if the existing functionality works as is you shouldn't change it

Reply:
the SV pipeline is also using annotate_globals so I'm even more confused by this change

Reply:
The change is to get rid of the extra global fields in the mitochondrial Matrix Table. The 'select_globals` does the same annotations as `annotate_globals` does. And it also does the global field selections. 
Let's see how a helper function works.